src/components/input/input.scss

```diff
@@ -99,7 +99,6 @@
 
 :host textarea,
 :host input {
-  transition: var(--calcite-animation-timing), height 0s;
```

Comment:
@macandcheese Can you double check this one? I had to remove this to avoid some flickering going on when the focus outline got displayed. This was introduced [here](https://github.com/Esri/calcite-components/commit/533c270ec2e22ba1db04261189fc29bd3e7bfc82#diff-4185271e827d2b26e19a1a16201fb3dc4a31448b796c61107967bb5cfe2ca722R65).

Reply:
Disregard the above. The flickering was caused by `outline-offset` being transitioned. I've explicitly prevented it from being animated on focusable elements where the transition property default is `all`.

Reply:
Yep, IIRC the height was there to prevent the input of type textarea from appearing laggy when resizing vertically, so as long as that doesn’t happen 👍